packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -1569,11 +1569,17 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         }
     }
 
+    private maybeAlias(id: string): string {
```

Comment:
`getInternalId` might be better as it returns an id. The param can be called `maybeAlias`?